### ISSUE TITLE
Error while installing Botkit

### ISSUE THREAD

**Issue by kbose02** [howdyai/botkit#1567]
Hi, I am getting below error when i try to install botkit

C:\Users\XXX>npm install -g botkit
C:\Users\XXX\AppData\Roaming\npm\botkit -> C:\Users\XXX\AppData\Roaming\npm\node_modules\botkit\bin\cli.js
+ botkit@0.7.2
updated 1 package in 57.87s

C:\Users\XXX>botkit new
√ Name your bot: ... mybot
√ Choose platform: » web
Installing Botkit...
An error occurred. You may already have that starter kit installed.
Error:Error: Command failed: git clone https://github.com/howdyai/botkit-starter-teams.git mybot&& cd mybot && npm install
'git' is not recognized as an internal or external command,
operable program or batch file.

**Comment by benbrown** [comment]
Do you have git installed on your machine?

**Comment by kbose02** [comment]
Yes...The env path for this aslo set under system properties. I have verified the path is correct "C:\Program Files\Git\cmd" and "C:\Program Files\Git\bin\"
